### PR TITLE
Updating title to Hacking Open Source

### PR DESCRIPTION
**Pull request by narcisacodreanu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <body>
         <div class="section" id="plan">
-                <h1>Practicing Git</h1>
+                <h1>Hacking Open Source at Columbia</h1>
```
Comment (narcisacodreanu) [inline index.html:12]: This doesn't make sense...it should be Hacking Open Source at Columbia featuring OpenHatch. Maybe @jcf2167 can take a look!

**Comment by jcf2167** [comment]
Yes!

File: index.html (hunk 1 above)
Comment (jcf2167) [inline index.html:12]: yes!

**Comment by narcisacodreanu** [comment]
closes #10

**Pull request merged**